### PR TITLE
Fix cert-generation, add permission to get secret

### PR DESCRIPTION
**Pull request by batas**
Cert generator is unable to get secret because of missing permission.

### CODE REVIEW

**Comment by functicons** [comment]
@batas Could you sign the CLA?

**Comment by batas** [comment]
CLA signed

**Comment by functicons** [comment]
@batas You need to reply `@googlebot I signed it!`.

**Comment by functicons** [comment]
Are you using different emails for your GitHub account and the commit? They need to be the same.

**Comment by batas** [comment]
Of course, I fixed it.

File: helm-chart/flink-operator/templates/generate-cert.yaml [hunk 1]
```diff
@@ -115,6 +115,33 @@ data:
...
+  - ""
+  resources:
+  - secrets
+  verbs:
+  - get
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: flink-operator-cert-manager
```
Comment (hongyegong) [inline helm-chart/flink-operator/templates/generate-cert.yaml:134]: Can we call this differently? We are not using cert-manager here.
Otherwise LGTM.

**Comment by batas** [comment]
Fixed

**Review by hongyegong — APPROVED** [review]

**Comment by hongyegong** [comment]
FYI, there is an ongoing [PR](https://github.com/GoogleCloudPlatform/flink-on-k8s-operator/pull/348#issue-504263325) aiming to move the name of `flinkOperatorNamespace` a level down below, so here might become `flinkOperatorNamespace.name` after the other PR is merged.

**Comment by mmazek** [comment]
What's up with all the PRs failing checks? I've looked into the logs and it does not seems as something related with the PR in particular - more like CI/CD setup issue. Can someone look into that?

**Comment by rogaha** [comment]
ping

**Pull request closed without merging**